tsl/src/remote/connection.c

```diff
@@ -455,6 +455,11 @@ extract_connection_options(List *defelems, const char **keywords, const char **v
 		}
 	}
 
+	/* If a user was not found in the connection options, assume the current
+	 * user */
```

Comment:
Since it is a proper sentence and starting with capitalized letter, it is good to end with a dot.
```suggestion
	 * user. */
```